scripts.sql

```diff
@@ -0,0 +1,9 @@
+select * from student
+WHERE age BETWEEN 10 AND 20;
+SELECT name FROM student;
+select * from student
+WHERE name LIKE '%Г%';
+SELECT * FROM student
+WHERE age < 18;
```

Comment:
```suggestion
WHERE age < id;
```
по заданию так требовалось